vignettes/programming_strategy.Rmd

```diff
@@ -374,25 +374,8 @@ where the labels are stored.
 
 # Unit Testing
 
-A function requires a set of unit tests to verify it produces the expected result. 
-If a function is defined in a file named `R/derive_baseline.R` then its accompanying unit tests should be stored in `tests/testthat/test-derive_baseline.R`.
-
-The testing framework to be used is [testthat](https://testthat.r-lib.org/).
-
-Unit tests should cover the functionality of the function.
-If another function g is called within a function f, the unit tests of f should not test the functionality of g. 
-This should be tested by the unit tests of g, i.e. unit tests should be added at the lowest level.
```

Comment:
I think we lost this part. Was this intended?

Reply:
agree - i would keep as things like the naming convention are important reminders here